pkg/controllers/cloud/controller_sync_status_test.go

```diff
@@ -184,79 +185,82 @@ var _ = Describe("Controller sync status", func() {
 			val := ctrlStatus.IsControllerSynced(ControllerTypeEE)
 			Expect(val).Should(BeFalse())
 		})
-		It("VirtualMachine Start with no CRs", func() {
-			vmReconciler := &VirtualMachineReconciler{
-				Log:    logf.Log,
-				Client: fakeClient,
-			}
-			ctrlStatus.SetControllerSyncStatus(ControllerTypeEE)
-			// Start VM reconciler with 0 CRs, and verify sync status, pendingSyncCount and initialization status.
-			err := vmReconciler.Start(context.TODO())
-			Expect(err).Should(BeNil())
-			Expect(vmReconciler.pendingSyncCount).Should(Equal(0))
-			Expect(vmReconciler.Initialized).Should(BeTrue())
-			val := ctrlStatus.IsControllerSynced(ControllerTypeVM)
-			Expect(val).Should(BeTrue())
+		// TODO: Enable the tests only VM start is implemented completely.
```

Comment:
Remove this.. Add TODO: If test needs to be re-added later

Reply:
Done